### PR TITLE
Update links - header and footer ones for website and docs & enhance UI

### PR DESCRIPTION
**Pull request by IsAmrish**
This PR will
- Fix footer for documentation and fix layout styling for documentation content part
- Update headers and footers links in documentation
- Update docs link in website
- Remove `contact us` section from footer
- Fix favicon issue in documentation
- Add top contributors section in footer of website
- Fix docs header -- basically adding padding in left and right
- Add font - Monteserrat-Medium for header links of docs

fixes: #93 

---
Screenshots of top contributors

Screenshot 2021-06-04 at 5 50 46 PM

desktop

### CODE REVIEW

**Comment by IsAmrish** [comment]
**Docs header**

docs-intro


**Docs footer**

Screenshot 2021-06-04 at 6 19 22 PM

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -42,34 +42,34 @@ module.exports = {
         },
         {
           label: "Community",
-          link: "https://openebs.io/community",
+          link: "https://gallant-pasteur-5e438c.netlify.app/community",
```
Comment (AVRahul) [inline docs/docusaurus.config.js:45]: one query is it possible to do something like this ??
```suggestion
          link: `${url}/community`,
```

File: website/src/constants/index.tsx [hunk 2]
```diff
@@ -19,6 +19,10 @@ export enum EXTERNAL_LINKS {
   SUBSCRIBE_NEWSLETTER='https://lists.cncf.io/g/cncf-openebs-announcements',
 };
 
+export const API = {
+  GITHUB_CONTRIBUTORS: 'https://api.github.com/repos/openebs/openebs-docs/contributors?q=contributions&order=desc'
```
Comment (rakeshPRaghu) [inline website/src/constants/index.tsx:23]: Should we take the github username and reponame from constants?

**Review by rakeshPRaghu — APPROVED** [review]

File: docs/src/components/Footer/index.js [hunk 3]
```diff
@@ -1,42 +1,169 @@
...
+const EmailSubscription = () => {
+  return (
+    <>
+      <span className="footer__title">
+        Stay in the know with our newsletter
+      </span>
+      <div className="margin-vert--md">
+        <a
+          className="doc-button doc-button-primary doc-button-curved  doc-button-lg"
+          href="https://lists.cncf.io/g/cncf-openebs-announcements"
```
Comment (satyapriyamishra222) [inline docs/src/components/Footer/index.js:25]: Let's bring it from external links config.

File: docs/src/components/Footer/index.js [hunk 4]
```diff
@@ -48,129 +175,81 @@ const SocialMedia = () => {
...
-    return () => {
-      setItems([]);
-    };
-  }, []);
+  const privacyPolicy = siteConfig?.customFields?.footbarLinks?.privacyPolicy;
 
+  return (
+    <div className="footer__bottom row margin-top--md col col--12">
+      <div className="margin-right--lg">
+        <span>{copyRight && copyRight}</span>
```
Comment (satyapriyamishra222) [inline docs/src/components/Footer/index.js:186]: What is the intention here ?

File: docs/src/components/Footer/index.js [hunk 5]
```diff
@@ -48,129 +175,81 @@ const SocialMedia = () => {
...
         </div>
       </div>
     </footer>
   );
 };
+
+export const Footer = () => {
+  const { width } = useViewport() || 0;
+  
+  return width < 996 ? <FooterMobile /> : <FooterWeb />;
```
Comment (satyapriyamishra222) [inline docs/src/components/Footer/index.js:254]: Can we bring this from a const saying if MobileView or DesktopView ?

File: docs/src/theme/DocItem/index.js [hunk 6]
```diff
@@ -75,15 +77,18 @@ function DocItem(props) {
                     <header>
                       <h1 className={styles.docTitle}>{title}</h1>
                     </header>
-                    <div className="action-buttons">
-                      <EditThisPage editUrl={editUrl} />
-                      <VersionDropdown />
-                    </div>
+                    {(width > 767) && (
```
Comment (satyapriyamishra222) [inline docs/src/theme/DocItem/index.js:80]: Let's bring this number from a const.

File: docs/src/theme/SearchBar/index.js [hunk 7]
```diff
@@ -91,7 +93,7 @@ const Search = props => {
       <input
         id="search_input_react"
         type="search"
-        placeholder="Search Documentation"
+        placeholder={`${width < 767 ? "Search" : "Search Documentation"}`}
```
Comment (satyapriyamishra222) [inline docs/src/theme/SearchBar/index.js:96]: Same here.

File: website/src/constants/index.tsx (hunk 2 above)
Comment (IsAmrish) [reply website/src/constants/index.tsx:23]: it's already in `constants` file :)

**Review by AVRahul — APPROVED** [review]

**Review by satyapriyamishra222 — APPROVED** [review]

**Review by rakeshPRaghu — APPROVED** [review]

**Pull request merged**